app/src/main/java/org/mifos/mobile/utils/Utils.kt

```diff
@@ -77,4 +81,11 @@ object Utils {
         }
         return builder.toString()
     }
+
+    fun hideSoftKeyboard(activity: Activity) {
```

Comment:
I believe, we already have this method implemented in BaseActivity, use that one, please.

Reply:
Ops that's totally my fault, I'll be extra considerate of existing code now onwards. Thanks @jawidMuhammadi. I've made the required changes.